plugins/declarative/src/voicecallmanager.h

```diff
@@ -76,7 +76,7 @@ public Q_SLOTS:
     void dial(const QString &msisdn);
     void dial(const QString &provider, const QString &msisdn);
 
-    void playRingtone();
+    void playRingtone(const QString &ringtonePath = QString());
```

Comment:
Just thinking that while changes like this retain source code compatibility, they are not binary compatible - which in general is more or less nasty. But should we care in this case? Like by perhaps having two methods (with and without arg)?

Reply:
We don't make promises for this component, and besides this is the declarative plugin so there is no such thing as binary compatibility here.